components/eventing-controller/cmd/eventing-controller/main.go

```diff
@@ -33,9 +34,15 @@ func main() {
 	setupLog := ctrl.Log.WithName("setup")
 
 	var metricsAddr string
+	var probeAddr string
+	var readyEndpoint string
+	var healthEndpoint string
 	var resyncPeriod time.Duration
 	var enableDebugLogs bool
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
+	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
```

Comment:
(nit): Although the below description is copied from the controller-runtime, it seems complete yet concise to me. 
`is the TCP address that the controller should bind to for serving health probes`

Reply:
Done, and matched the metrics-address description as well